From 73e1d130538ea4cdca2dfd6a273cb3f9ac092934 Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Tue, 26 Jun 2018 11:53:22 -0700
Subject: [PATCH 1/3] DNF 3: config substitutions moved from dnf to libdnf

This just moved from dnf.conf to libdnf.conf, no big problem.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 pyanaconda/payload/dnfpayload.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/payload/dnfpayload.py b/pyanaconda/payload/dnfpayload.py
index dee3903eb..8b8be68c9 100644
--- a/pyanaconda/payload/dnfpayload.py
+++ b/pyanaconda/payload/dnfpayload.py
@@ -54,7 +54,7 @@ import dnf.logging
 import dnf.exceptions
 import dnf.repo
 import dnf.callback
-import dnf.conf.parser
+import libdnf.conf
 import dnf.conf.substitutions
 import rpm
 import librepo
@@ -325,7 +325,7 @@ class DNFPayload(payload.PackagePayload):
         Currently supports $releasever and $basearch.
         """
         if url:
-            return dnf.conf.parser.substitute(url, self._base.conf.substitutions)
+            return libdnf.conf.ConfigParser.substitute(url, self._base.conf.substitutions)
 
         return None
 
-- 
2.18.0.rc2

